src/SavedSearch.php

```diff
@@ -1195,6 +1219,31 @@ public function setEntityRecur(array $ids, $eid, $recur)
         return $result;
     }
 
+    /**
+     * Set visibility from massive actions
+     *
+     * @param array   $ids   Items IDs
+     * @param integer $is_private   VIsibility
+     *
+     * @return boolean
+     */
+    public function setVibility(array $ids, $is_private)
```

Comment:
```suggestion
     * @param bool $is_private   Visibility
     *
     * @return boolean
     */
    public function setVibility(array $ids, bool $is_private)
```

Maybe the update value should be adjusted, to be tested.

A unit test must be added for this new method anyway.